ui/client/components/toolbars/AttachmentsPanel.tsx

```diff
@@ -0,0 +1,14 @@
+import {CollapsibleToolbar} from "../toolbarComponents/CollapsibleToolbar"
+import ProcessAttachments from "../ProcessAttachments"
+import React from "react"
+import {useTranslation} from "react-i18next"
+
+export function AttachmentsPanel() {
+  const {t} = useTranslation()
+
+  return (
+    <CollapsibleToolbar id="ATTACHMENTS-PANEL" title={t("panels.attachments.title", "Attachments")}>
```

Comment:
why capital letters?

Reply:
to differentiate from string that should be translated and avoid eslint-disable